README.md

```diff
@@ -2,21 +2,30 @@
 
 [![Greenkeeper badge](https://badges.greenkeeper.io/probot/autolabeler.svg)](https://greenkeeper.io/)
 
-> a GitHub App built with [probot](https://github.com/probot/probot) that adds labels to Pull Requests based on matched file patterns.
+> A GitHub App built with [probot](https://github.com/probot/probot) that adds labels to Pull Requests based on matched file patterns.
 
 ## Using
 
-Configure by creating a `.github/autolabeler.yml` file with a yaml file in the format of `label: file/path`. 
+Configure by creating a `.github/autolabeler.yml` file with a [yaml file](https://en.wikipedia.org/wiki/YAML) in the format of `label: file/path`. Then add the [Probot Auto Labeler Bot](https://github.com/apps/probot-autolabeler) to your repository.
 
 For example,
+
 ```yaml
 frontend: ["*.js", "*.css", "*.html"]
 backend: ["/app", "*.rb"]
 legal: ["LICENSE*", "NOTICES*"]
 config: .github
 ```
+
 Then if a pull request is opened that has `scripts/widget.js` modified, then the frontend label will be added.
 
+You can see what it looks like when working below;
+![Probot Autolabeler Bot adding labels to a pull request](docs/working.png)
```

Comment:
I don't think a screenshot is necessary, since the relationship of pushing code and applying labels isn't something that can be easily captured in an image.

Reply:
The screenshot shows the bot in operation pretty well IMHO. It has multiple `probot-autolabler [bot] added label XXX on YYY` lines which shows how as you update the pull request the bot will add new labels and stuff.

Reply:
IMHO a screenshot indeed it helpful, but the current one doesn't make the relationship between the commits and the bot's activities clear. Maybe a dummy example with commit messages that relate directly to the probot's actions (e.g. "add file foobar" --> probot added label foobar) would be more helpful here.

Reply:
I've removed the screenshot for now.